WinImageBuilder.psm1

```diff
@@ -39,6 +41,78 @@ $VirtIODriverMappings = @{
 
 . "$scriptPath\Interop.ps1"
 
+
+function Get-AvailableConfigs {
+    $availableConfigs = @()

```

Comment:
This logic (Get-AvailableConfigs,Get-ConfigFromTemplate, Get-GlobalConfigs) can be all moved in a Configs.psm1 file and imported afterwards.